templates/cloudsshkey.j2

```diff
@@ -22,6 +22,17 @@ keys_received=0
 file_count=0
 error_count=0
 
+case `uname -s` in
```

Comment:
is there any reason not to use a jinja2 filter and `ansible_os_family` to create a script for each platform?

Reply:
fixed in b94fa4df0